content/webapp/components/SearchResults/SearchResults.tsx

```diff
@@ -49,7 +49,10 @@ const SearchResults: FunctionComponent<Props> = ({
     {items.map(
       (item, index) =>
         item.type !== 'card' && (
-          <Result key={item.id}>
+          <Result
+            key={item.id}
+            data-testid={index === 0 ? 'search-result' : undefined}
```

Comment:
I thought there was a typo here but noticing now we have both `data-testid` (although only once) and `data-test-id`. Doesn't really matter but we could align?

Reply:
I didn't know about `getByTestId`! Sounds like a nice thing to make use of!

Reply:
I started with `data-test-id` but I didn't think Playwright was picking up on it. It's possible I had that wrong though and something else was preventing the test from passing before – let me check. Seems like this is the [default for Playwright](https://playwright.dev/docs/locators#set-a-custom-test-id-attribute) but it's configurable

Reply:
I'm not against going with the default tbh, but am thinking we should defo make use of the helper function across! TIL 😁

Reply:
I think the philosophy is that using `data-test-id` should be a last resort and preferably you'd select by something a user can actually see. But sometimes you need it as an escape hatch

Reply:
Oh absolutely agree! I meant where we do lookups like `.find("[data-test-id='TextInputErrorMessage']"))`, we could look at using `getByTestId` instead, just didn't know we could :)

Reply:
thanks for trying! but yeah don't let that block this, I shared with Adi too and we'll aim to use that moving forward!